inc/functions.php

```diff
@@ -595,7 +620,7 @@ function preferred_languages_pre_load_script_translations( $translations, $file,
 			$translations_json     = json_decode( $translations, true );
 
 			foreach ( $translations_json['locale_data']['messages'] as $key => $translation ) {
-				if ( isset( $all_translations_json['locale_data']['messages'][ $key ] ) ) {
+				if ( ! empty( array_filter( (array) $all_translations_json['locale_data']['messages'][ $key ] ) ) ) {
```

Comment:
`$all_translations_json['locale_data']['messages'][ $key ]` could be something like `array( [0] => '' )`, hence this changed check.